src/utils/queries.ts

```diff
@@ -1,17 +1,40 @@
 import { ethers } from "ethers";
+import { FilterOptions } from "src/types";
 
 const optionalAddressFilter = (address?: string) =>
-  address ? `where: { owner: "${address.toLowerCase()}" }` : "";
+  address ? `owner: "${address.toLowerCase()}"` : "";
 
-export const getBetsQuery = (
-  limit: number,
-  skip: number,
-  address?: string
-) => `{
+const optionalFilterOptions = (filter?: FilterOptions) => {
+  switch (filter) {
+    case "ALL_BETS":
+      return "";
+    case "PENDING":
+      // TODO: filter here instead of after formatBetHistory when subgraph is updated
```

Comment:
include in comment -- "when subgraph is updated _with market oracle as data source_"